tests/pact/strings.repl

```diff
@@ -0,0 +1,15 @@
+;; strings.repl : tests for ops on strings
+
+"===== str-to-list"
+(expect "str-to-list on str returns a list of single char strings" 
+  ["a" "b" "c"] (str-to-list "abc"))
+(expect-failure "str-to-list fails on list" (str-to-list []))
```

Comment:
Generally it's good to use the other `expect-failure` overload to ensure the failure is expected, it matches on `isInfixOf` the error string so you can keep it short

Reply:
Fixed. Thank you!